### PR TITLE
Fix indentation for consistency

### LINKED ISSUE #3252: Bad caption formatting on "Pause your code with breakpoints"
**Describe the bug**
Figure 8 caption markdown formatting not displaying correctly, due to indentation.

**To Reproduce**
1. Go to [Exception breakpoints](https://developer.chrome.com/docs/devtools/javascript/breakpoints/#exceptions)
2. Scroll down to Figure 8
3. See asterisks around the caption of figure 8.

**Expected behavior**
Text surrounded by two asterisks in markdown should be bold.

**Screenshots**
Screenshot from 2022-07-18 14-33-41

**Desktop:**
 - OS: Ubuntu 20.04.4 LTS
 - Browser: Chrome
 - Version Version 103.0.5060.114 (Official Build) (64-bit)

**Additional context**
I've fixed this in pull request #3251

### PR DESCRIPTION
**Pull request by andyglover**
Fix #3252  - figure 8 caption formatting for consistency and proper display on web

https://deploy-preview-3251--developer-chrome-com.netlify.app/docs/devtools/javascript/breakpoints/#exceptions

### CODE REVIEW

**Comment by heyawhite** [comment]
While this fixes the markdown issue, it resets the count on the next step below: 

Step 3 becomes 1

I'm going to see if I can switch it to HMTL to get the style to work without breaking the ordered list

File: site/en/docs/devtools/javascript/breakpoints/index.md [hunk 1]
```diff
@@ -171,8 +171,9 @@ uncaught exception.
     {% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/Ko2qtw41xR12rOnCfQzt.png", alt="Pause on exceptions", width="28", height="28" %}.
     It turns blue when enabled.
 
-    {% Img src="image/admin/Rc3ID0gnnhS8JBMCbF6J.png", alt="The 'Pause on exceptions' button.", width="800", height="594" %}
-    **Figure 8**: The **Pause on exceptions** button
+{% Img src="image/admin/Rc3ID0gnnhS8JBMCbF6J.png", alt="The 'Pause on exceptions' button.", width="800", height="594" %}
+
+**Figure 8**: The **Pause on exceptions** button
```
Comment (heyawhite) [inline site/en/docs/devtools/javascript/breakpoints/index.md:176]:
```suggestion
   {% Img src="image/admin/Rc3ID0gnnhS8JBMCbF6J.png", alt="The 'Pause on exceptions' button.", width="800", height="594" %}

   <strong>Figure 8</strong>: The <strong>Pause on exceptions</strong> button
```

File: site/en/docs/devtools/javascript/breakpoints/index.md [hunk 2]
```diff
@@ -171,8 +171,9 @@ uncaught exception.
     {% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/Ko2qtw41xR12rOnCfQzt.png", alt="Pause on exceptions", width="28", height="28" %}.
     It turns blue when enabled.
 
-    {% Img src="image/admin/Rc3ID0gnnhS8JBMCbF6J.png", alt="The 'Pause on exceptions' button.", width="800", height="594" %}
-    **Figure 8**: The **Pause on exceptions** button
+   {% Img src="image/admin/Rc3ID0gnnhS8JBMCbF6J.png", alt="The 'Pause on exceptions' button.", width="800", height="594" %}
+
+   <strong>Figure 8</strong>: The <strong>Pause on exceptions</strong> button
```
Comment (heyawhite) [inline site/en/docs/devtools/javascript/breakpoints/index.md:176]:
```suggestion
   <figure>{% Img src="image/admin/Rc3ID0gnnhS8JBMCbF6J.png", alt="The 'Pause on exceptions' button.", width="800", height="594" %}
   <figcaption><strong>Figure 8</strong>: The <strong>Pause on exceptions</strong> button</figcaption>
   </figure>
```

**Review by andyglover — COMMENTED** [review]
Nice, thank you for catching that it reset the numbering! Well done :)

**Comment by heyawhite** [comment]
Okay we got it! Stylistically it looks different from other places in the doc. However, other figures are not included within multiple steps, they're at the end of steps. 

Could be that every figure is put in the `<figure>Image<figcaption>Description</figcaption></figure>` configuration, but it doesn't overly matter.

@andyglover I'm happy to approve and merge as is, but if you want to make that unification, go for it!

**Review by heyawhite — APPROVED** [review]

**Comment by andyglover** [comment]
Well done @heyawhite, your changes look great as is! I appreciate your help!

**Pull request merged**